lib/prometheus/engineering_collector.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+unless defined? Rails
+  require File.expand_path('../../config/environment', __dir__)
+end
+
+# rubocop:disable Metrics/MethodLength
+# rubocop:disable Metrics/AbcSize
+# Collects stats relevent for the engineers from the database
+# when /metrics is called. This runs directly on prometheus_exporter.
+class EngineeringCollector < PrometheusExporter::Server::TypeCollector
```

Comment:
:see_no_evil: I took the opportunity to add an engineering collector for prometheus. We should use this for any info that we'd like to collect about the health/integrity of our data set. These first metrics are all related to detecting records that reference some missing (i.e. deleted) host ID.